### DISCUSSION TITLE
Welcome to Epiverse-TRACE Discussions!

### DISCUSSION THREAD

**Discussion by Bisaloo** [epiverse-trace/epiverse-trace.github.io#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

Key links:
* Join the discussion at https://github.com/orgs/epiverse-trace/discussions
* How to post an R code question using [`{reprex}`](https://reprex.tidyverse.org/) by Applied Epi ([link to post on their community platform](https://community.appliedepi.org/t/how-to-post-an-r-code-question/623))
* All Epiverse-TRACE materials for learning and teaching are free to reuse: https://epiverse-trace.github.io/learn.html

➡️ To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shola-B** [comment]
Bonjour! Shola Able-Thomas here. Great training!

**Comment by AdannaUg** [comment]
Hi, my name is Adanna from Nigeria. Glad to be part of this team. Cheers.

**Comment by seyalhagiepapa** [comment]
Hello, Am Alhagie Papa Say from The Gambia. I really enjoyed the training.

**Reply by Dnyamwaya** [reply]
Hello, Ass.Prof. Doris at LSHTM . Happy to be here

**Comment by Sanyang-maker** [comment]
Hello my name is Armando from the Gambia. I am  happy to be here

**Comment by Fatoumata-SECK** [comment]
As'Salam AleyKum... This is Fatoumata SECK, from Senegal.
Happy to be here 😍

**Comment by ErnestMupenziBIZI** [comment]
Hello, 

This is Ernst From Rwanda, A participants in the Epiverse trace pre-summit training 2024. Happy to be here

**Comment by massa-atsu** [comment]
Hello
I am Joshua Atsu from Ghana. I am happy to join this discussion.
Happy collaborating.

**Comment by Sainey1992** [comment]
Hello. Happy to be here. Great to be part of the training.

**Comment by Elpastore** [comment]
Hi, my name is Abdoulaye Sadio from Senegal, happy to be part of this wonderful training!

**Comment by sbaptist** [comment]
Hi! I am Sang B. Mendy and I am from The Gambia. I am so happy to be part of this training.

**Comment by regineforte** [comment]
Hi, my name is Bayana Ndoba from Congo -Kinshasa and I'm happy to be a part of Epiverse family

**Comment by danibiftu** [comment]
Hello, I am Daniel from Ethiopia. I am happy to be a member of this community.

**Comment by Aji05-do** [comment]
Hi, I’m Aji fanna Boye from The Gambia.
I’m really happy to be part of this community and hoping to learn more from this amazing community.

**Comment by iskode** [comment]
Hello,
I am Ismael KONE from Ivory Coast (Côte d'Ivoire)
I'm attending the epiverse-TRACE training. 
The tools you are building are phenomenal !!!
Big Shout out !
Each public health ministry should have representative learning this
tool !

**Comment by borison365** [comment]
Hi! everyone, I am Boris, just finished the Epiverse trace training course in The Gambia, 

Looking forward to collaborating with you!

**Comment by pitisolim** [comment]
Hello guys, I'm  Solim PITIKALO Ffrom TOGO,I'm happy to be here.

**Comment by Edwardirene199** [comment]
Hi everyone

I am Irene Edward from Tanzania. Happy to be here

**Comment by Paupauash** [comment]
Hello everyone. My name's Paustella Azokpota. More than happy to join.

**Comment by KebbaJ** [comment]
hello guys, I'm Kebba Jobarteh from The Gambia. Glad to collaborate with you all

**Comment by Maherjerry** [comment]
Hello, my name is Muhammed Bah from The Gambia. I am happy to be part of the Epiverse training and it is a cool initiative.

**Comment by gambarry** [comment]
Hello everyone, I am Barry from the Gambia.

I am so excited to be here. Cheers to coding!

**Comment by MusclineGanda** [comment]
Hello, I'm Muscline Ganda from Zimbabwe, I'm honored to have been part of this transformative and incredible  Epiverse-Trace training on Outbreak Analytics and Applied Modelling in R. It was a training worth attending .

**Comment by saniamadouabdou** [comment]
Hello, my name is Sani from Niger. I am excited to attend the Epiverse-Trace Pre-Summit Training and want to thank the team for the amazing work in building tools and knowledge to advance epidemiological research.

**Comment by SokhnaFatimahBigue** [comment]
Hello, my name is Sokhna Fatimata Bigué fromSenegal and I'm happy to be a part of Epiverse family.

**Comment by kornelius14** [comment]
Hello, I am Kornelius from Indonesia, and I am extremely happy to be here.

**Comment by jnamulondo** [comment]
Hello, Joyce from Uganda. Thank you for this training. I am excited to be part of this community.

**Comment by Nantambi** [comment]
Hello, am Faith from Uganda. I am happy to be part of this training. Thank you for the wonderful training.

**Comment by arthurshem86** [comment]
Halo, 
Arthur here!!...It was a pleasure connecting and learning from the Epiverse-TRACE team in Uganda.

**Comment by kornelius14** [comment]
Hi there. Could someone please clarify this for me? I had previously used the socialmixr package without any issues using socialmixr::get_survey("https://..."). However, during tonight’s refresher session, when I tried the same command:

list_surveys()
zimbabwe <- socialmixr::get_survey(survey = "https://doi.org/10.5281/zenodo.3886638")

I encountered this error:
**Error in if (is.character(txt) && length(txt) == 1 && nchar(txt, type = "bytes") < : missing value where TRUE/FALSE needed**. Thank you for your help.

**Reply by avallecam** [reply]
Thanks @kornelius14 for asking this separately, too. We answered it here https://github.com/orgs/epiverse-trace/discussions/400

**Comment by Enwerem-K** [comment]
Hello, my name is Kenneth Enwerem aka the R Mallam - the Epiverse Fellow for Nigeria. It's great to be here

**Comment by joshuaaniaku** [comment]
Hi everyone, I'm Joshua Aniaku, Epiverse fellow for Ghana. Glad to meet you all

**Comment by M-chep** [comment]
Hello,
Mercy Cheptoo from Kenya, Epiverse Data Science Fellow.
Pleasure connecting in Senegal.

**Comment by mardochee01** [comment]
Hello, everyone

**Comment by VulstanShedura** [comment]
Hello

**Comment by bmolleh255** [comment]
Hello,

My name if Bailah Molleh. I am the Epiverse Fellow from Sierra Leone.

**Comment by mardochee01** [comment]
Hello,
Mardochée GNERAN from Côte d'Ivoire, Epiverse Data Science Fellow.

**Comment by degninou** [comment]
> ## 👋 Welcome!
> 
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
>     * Ask questions you’re wondering about.
> 
>     * Share ideas.
> 
>     * Engage with other community members.
> 
>     * Welcome others and are open-minded. Remember that this is a community we build together 💪.
> 
> 
> Key links:
> 
>     * Join the discussion at https://github.com/orgs/epiverse-trace/discussions
> 
>     * How to post an R code question using [`{reprex}`](https://reprex.tidyverse.org/) by Applied Epi ([link to post on their community platform](https://community.appliedepi.org/t/how-to-post-an-r-code-question/623))
> 
>     * All Epiverse-TRACE materials for learning and teaching are free to reuse: https://epiverse-trace.github.io/learn.html
> 
> 
> ➡️ To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Greetings @Degoot-AM, this is Dègninou from Togo. Great presentation at the Epiverse Fellowship Worshop, WHO Hub Dakar.

**Comment by avallecam** [comment]
Hi all! 

We are conducting a brief survey to assess the usage of the Epiverse-TRACE packages.

Go and find more about it on this new Discussions entry >> #417

**Comment by basil4life123** [comment]
Hi, my name is Basil Izuchukwu.

I'm happy to be a part of this great community. 

We will stay in touch, **codedly** ☺

**Comment by gemechu219-stack** [comment]
Hi, this is Gemechu from Ethiopia. I am happy to be part of this network.

**Comment by s1576767** [comment]
Hi,
Nay from Lao PDR, working in the Health Emergencies team, and proud to be a part of the group. Data enthusiast and very impressed with the packages; I look forward to using these resources in my daily work.
